Description: Use rm -f in make clean
 This patch uses the -f option for rm in make clean so that no error occurs
 when make clean is run while no object or binary files exist.  (It also adds
 a newline at the end of the Makefile thanks to emacs.)
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: <2014-07-21>

--- wmtime-1.1.orig/wmtime/Makefile
+++ wmtime-1.1/wmtime/Makefile
@@ -17,9 +17,9 @@ wmtime: $(OBJS) $(XPMS)
 
 clean::
 	for i in $(OBJS) ; do \
-		rm $$i; \
+		rm -f $$i; \
 	done
-	rm wmtime
+	rm -f wmtime
 
 install:: wmtime
-	install -m 755 wmtime $(DESTDIR)/usr/bin
\ No newline at end of file
+	install -m 755 wmtime $(DESTDIR)/usr/bin
